bootstrap/scripts/plugin.py

```diff
@@ -11,6 +11,17 @@
 import validation
 
 
+def basename(value: str) -> str:
+    return Path(value).name
+
+
+def talos_patches(value: str) -> list[str]:
+    path = Path(f'bootstrap/templates/kubernetes/bootstrap/talos/patches/{value}')
```

Comment:
@wasurerarenai should this be `bootstrap/templates/kubernetes/bootstrap/talos/patches` or `kubernetes/bootstrap/talos/patches` ?